src/main/java/org/apache/commons/release/plugin/mojos/CommonsSiteCompressionMojo.java

```diff
@@ -147,8 +148,9 @@ private void getAllSiteFiles(final File siteDirectory, final List<File> filesToC
      */
     private void writeZipFile(final File outputDirectory, final File directoryToZip, final List<File> fileList)
             throws IOException {
-        try (FileOutputStream fos = new FileOutputStream(outputDirectory.getAbsolutePath() + "/site.zip");
-                ZipOutputStream zos = new ZipOutputStream(fos)) {
+        try (OutputStream fos = Files.newOutputStream(new File(outputDirectory.getAbsolutePath() + "/site.zip")
+                .toPath());
+             ZipOutputStream zos = new ZipOutputStream(fos)) {
```

Comment:
Nit-picking, but I think there's one extra space before `ZipOutputStream`, or at least looking at the GitHub interface it appears to have an extra one.

Reply:
Hi @kinow 
I have check the code directly and i can't see the extra space. I also run the checkstyle and nothing :-(
Regards,